src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -457,7 +457,7 @@ export function ConfirmationModal ({ confirmationModalProperties, setConfirmatio
       actionParams = { id, amount, counterparty, notes }
       actionHook = payTransaction
       message = <>
-        Accept request for payment of {presentHolofuelAmount(amount)} TF from {counterparty.nickname || presentAgentId(counterparty.id)}?
+        Accept the request and send {counterparty.nickname || presentAgentId(counterparty.id)} {presentHolofuelAmount(amount)} TF?
```

Comment:
Should 'request' be capitalized here? It is in the wording located in te Version One ticket, so I just want to verify. (I know this is super nitpicky, but the focus of this ticket is rather detailed, so let's just double-check this here.)

Reply:
yeah I made a judgement call there. Didn't look right to me. I'll run it passed Rob before hitting complete on the ticket.

Reply:
If we're polling, I agree that it's not a proper noun, and therefore shouldn't be capitalized.